deepwell-core/src/models/page.rs

```diff
@@ -77,4 +79,17 @@ impl Page {
     pub fn exists(&self) -> bool {
         self.deleted_at.is_none()
     }
+   
+    #[cfg(feature = "ftml_compat")]
+    pub fn create_pageinfo<'a, TScoring: Scoring>(&self, votes: &Votes) -> ftml::PageInfoOwned {
+        ftml::PageInfoOwned {
+            title: String::from(self.title()),
+            alt_title: match self.alt_title() { None => None, Some(s) => Some(String::from(s)) },
+            tags: self.tags().to_vec(),
+            header: None,
+            subheader: None,
+            rating: TScoring::score(votes) as i32,
```

Comment:
This is not a safe cast. It doesn't really matter though, pull `ftml`, I updated the rating type to match deepwell-core